src/ExpressReceiver.ts

```diff
@@ -254,25 +254,26 @@ async function verifyRequestSignature(
 function parseRequestBody(
   logger: Logger,
   stringBody: string,
-  contentType: string | undefined) {
+  contentType: string | undefined): string | querystring.ParsedUrlQuery {
```

Comment:
i think the appropriate return type might actually be `any`. while it can be a `string` or a `querystring.ParsedUrlQuery`, it can also be the return type of `JSON.parse()` which is `any`. a union of `any` with anything is `any`, therefore the return type is actually `any`. i didn't make this change locally in my editor, but i'm kind of surprised the type checker isn't complaining about this.

Reply:
I'm also a bit surprised it didn't complain.